src/utils/analysis-utils.ts

```diff
@@ -524,15 +524,17 @@ export function createLegendFromFeatureArray(
     (a, b) => a - b, // This function is required since JS assumes unicode values.
   );
 
-  const statsLen = stats.length;
+  const uniqueStats = [...new Set(stats)]; // Remove repeated elements.
```

Comment:
Can you explain a bit more when this is used and why it is important to remove the repeated elements?

*Technically speaking, the values below are not quintiles anymore if you pre-filter to remove duplicate values.

Reply:
Yes I understand, however:
- these are not quintiles anymore by definition (not a huge deal - my request was mostly to add comments in the code)

- this problem is a bit more complex than it seems, and this solution might not work exactly as you expect. I recommend adding some tests for this function with a few edge cases. Eg. what happens if there are less than 5 unique values? Then we would have 1,734,633 multiple times as well. -> Using unique values also risks adding its own edge cases.
- We could also envision using an equal value split between the min/max for example.